Source/DiabloUI/selconn.cpp

```diff
@@ -54,27 +53,27 @@ void SelconnLoad()
 	vecSelConnDlg.push_back(std::make_unique<UiArtText>(_("Requirements:"), rect3));
 
 	SDL_Rect rect4 = { (Sint16)(PANEL_LEFT + 35), (Sint16)(UI_OFFSET_Y + 275), DESCRIPTION_WIDTH, 66 };
-	vecSelConnDlg.push_back(std::make_unique<UiArtText>(selconn_Description, rect4));
+	vecSelConnDlg.push_back(std::make_unique<UiArtText>(selconn_Description, rect4, UiFlags::FontSize12, 1, 16));
 
 	SDL_Rect rect5 = { (Sint16)(PANEL_LEFT + 30), (Sint16)(UI_OFFSET_Y + 356), 220, 31 };
-	vecSelConnDlg.push_back(std::make_unique<UiArtText>(_("no gateway needed"), rect5, UiFlags::AlignCenter | UiFlags::FontMedium));
+	vecSelConnDlg.push_back(std::make_unique<UiArtText>(_("no gateway needed"), rect5, UiFlags::AlignCenter | UiFlags::FontSize24));

```

Comment:
This text is just a little bit too wide so the last character ends up on its own line. Also, I'm not sure why this text is necessary in the first place.

Reply:
It was for selecting a bnet server originally, we could instead use it to let the player set port, bind ip or ZeroTier network.

Reply:
I just meant that I wasn't sure why we need it when we don't have a bnet server with separate gateways, but I guess you're saying that we're keeping it with thoughts of potentially repurposing it. I like those ideas you have for repurposing.